_posts/2020-07-12-ksf-lead-after-first-weekend.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: news
+title: Kenilworth Sixth Form lead after the first weekend of matches
+---
+
+After the first 24 matches, team KSF from Kenilworth Sixth Form head up the
+league leaderboard with X points. They're closely followed by JAMDynamics and
+Lawrence Sheriff School.
+
+The league matches provide a proving ground for all the teams' hard work over
+the past few months in preparing their robot code. There is also a prize for the
+rookie team which scores the best in the league, which the Kenilworth Sixth Form
+and JAMDynamics teams are currently well placed for.
+
+The teams now have a week to work on and improve their code before another 24
+league numbers are played over the course of next weekend. As we've seen with
+the changes in the league from Sunday's matches, this means there's still all to
+play for.
+
+If you would like to catch up on this weekend's livestreamed matches, they're
+available on Youtube:
+
+* [Saturday: League Session 1](https://youtu.be/xLL7SoQywf4)
+* [Sunday: League Session 2](https://youtu.be/7JoW4zXSeZE)
```

Comment:
I think it's worth embedding these like on the event pages

Reply:
I don't think these are emboldened on the event pages?
Compare to https://studentrobotics.org/events/sr2020/virtual-competition-league-2/